src/common/containers/UserDetail/index.jsx

```diff
@@ -83,6 +83,7 @@ function mapStateToProps(state, ownProps) {
 
   const userSummary = Object.values(userSummariesByUserId).find(userSummary => {
     return userSummary.user && (
+      userSummary.user.handle.toLowerCase() === identifier ||
       userSummary.user.handle === identifier ||
```

Comment:
No need for two checks; convert both `userSummary.user.handle` and `identifier` to lowercase and make one comparison.